bootstrap/ncpms_seed.py

```
@@ -0,0 +1,74 @@
+        all_vals = ",\n".join(query_parts)
+        query = f"""
+        INSERT INTO ncpms_diagnoses (pest_name, crop_name, ecology_info, biology_prvnbe_mth, chemical_prvnbe_mth, prevent_method, formatted_markdown)
+        VALUES
+        {all_vals}
+        ON CONFLICT (pest_name, crop_name) DO UPDATE SET
+            ecology_info = EXCLUDED.ecology_info,
+            biology_prvnbe_mth = EXCLUDED.biology_prvnbe_mth,
+            chemical_prvnbe_mth = EXCLUDED.chemical_prvnbe_mth,
+            prevent_method = EXCLUDED.prevent_method,
+            formatted_markdown = EXCLUDED.formatted_markdown;
+        """
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)